### ISSUE TITLE
Optional "species" field for both request and response

### ISSUE THREAD

**Issue by allasm** [MatchMakerExchange/mme-apis#39]
...for when a gene belongs to a mouse/zebra fish/etc. model. Should default to "human".

**Comment by fschiettecatte** [comment]
Here is the NCBI nomenclature for species along with scientific name and common name:

    8355:   'African clawed frog', 'Xenopus laevis'
    4754:   'Ascomycetes', 'Pneumocystis carinii'
    4932:   'Baker's yeast', 'Saccharomyces cerevisiae'
    9913:   'Cattle', 'Bos taurus'
    44689:  'Cellular slime molds', 'Dictyostelium discoideum'
    562:    'Enterobacteria', 'Escherichia coli'
    4896:   'Fission yeast', 'Schizosaccharomyces pombe'
    7227:   'Fruit fly', 'Drosophila melanogaster'
    3055:   'Green algae', 'Chlamydomonas reinhardtii'
    11103:  'Hepatitis C virus', 'Hepatitis C virus'
    10090:  'House mouse', 'Mus musculus'
    9606:   'Human', 'Homo sapiens'
    5833:   'Malaria parasite P. falciparum', 'Plasmodium falciparum'
    4577:   'Monocots', 'Zea mays'
    2104:   'Mycoplasma', 'Mycoplasma pneumoniae'
    6239:   'Nematodes', 'Caenorhabditis elegans'
    10116:  'Norway rat', 'Rattus norvegicus'
    4530:   'Rice', 'Oryza sativa'
    3702:   'Thale cress', 'Arabidopsis thaliana'
    31033:  'Torafugu', 'Takifugu rubripes'
    7955:   'Zebrafish', 'Danio rerio'

**Comment by fschiettecatte** [comment]
And the field should be called 'organism'.

**Comment by cmungall** [comment]
'organism' could be interpreted to be the specific organism instance. If the intent is to always place a taxon in there, how about 'taxon'. Horses for courses, not sure how much it matters so long as it's documented.

Is the field value an object or an ID? If an ID, I favour `ncbi_taxon_id` to make it unambiguous.

**Comment by buske** [comment]
All things being equal, how about following GA4GH's lead here? They suggest `"NCBITaxon:9606"` for representing human. Here's the relevant part of their file:
```
  /**
  The species of this individual. Using
  [NCBI taxonomy](http://www.ncbi.nlm.nih.gov/taxonomy) is recommended.
  For a representation of an NCBI Taxon ID as an OntologyTerm, see
  [NCBITaxon Ontology](http://www.obofoundry.org/wiki/index.php/NCBITaxon:Main_Page).
  For example, 'Homo sapiens' has the ID 9606. The NCBITaxon ontology ID for this is
  NCBITaxon:9606, which has the URI http://purl.obolibrary.org/obo/NCBITaxon_9606
  */
  union { null, OntologyTerm } species = null;
```

I included a basic version of this in PR #59, but will update that with people's consensus.

**Comment by fschiettecatte** [comment]
+1, the important thing for me is to use the NCBI taxonomy because it pervasive.
